crates/nu-protocol/src/engine/state_working_set.rs

```diff
@@ -24,10 +24,10 @@ pub struct StateWorkingSet<'a> {
     pub permanent_state: &'a EngineState,
     pub delta: StateDelta,
     pub external_commands: Vec<Vec<u8>>,
-    /// Current working directory relative to the file being parsed right now
-    pub currently_parsed_cwd: Option<PathBuf>,
-    /// All previously parsed module files. Used to protect against circular imports.
-    pub parsed_module_files: Vec<PathBuf>,
+    /// Absolute paths to scripts that are being processed.
+    /// The script currently being processed is on the top of the stack.
+    /// Circular import/sourcing is prevented by checking the stack before processing.
+    pub scripts: Vec<PathBuf>,
```

Comment:
Same, we should be accurate with our terminology (defering to @kubouch for the prime expertise on this part of the engine)

Reply:
Agreed, we have more files than scripts and the plain name “scripts” doesn't carry the intended meaning for the field (i.e., stack of accessed parsed files).

Reply:
Noted. I didn't think too much on the names initially, and I agree "file" is a much better name than "script".

Reply:
Same as above, I opted for the shorter name "files" hoping that the comments (and IDE tooltips) would be clear enough.